workers/unity/Packages/com.improbable.gdk.core/Worker/WorldCommandsReceivedStorage.cs

```diff
@@ -71,5 +83,114 @@ public void AddResponse(WorldCommands.EntityQuery.ReceivedResponse response)
         {
             return new ReceivedMessagesSpan<WorldCommands.EntityQuery.ReceivedResponse>(entityQueryResponses);
         }
+
+        ReceivedMessagesSpan<WorldCommands.CreateEntity.ReceivedResponse>
+            IDiffCommandResponseStorage<WorldCommands.CreateEntity.ReceivedResponse>.GetResponse(long requestId)
+        {
+            if (!createEntitySorted)
+            {
+                createEntityResponses.Sort(comparer);
```

Comment:
Is Sorting required before a `GetResponseIndex`? If so, shouldn't `GetResponseIndex` take care of that?

Reply:
Yes and it can but it checking if something is sorted is an O(n) operation so I'd rather not do it each time.

Reply:
This basically just does a binary search.